drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -3965,6 +3965,10 @@ RID RenderingDeviceVulkan::framebuffer_create(const Vector<RID> &p_texture_attac
 	for (int i = 0; i < p_texture_attachments.size(); i++) {
 		Texture *texture = texture_owner.get_or_null(p_texture_attachments[i]);
 
+		if (texture && texture->layers != p_view_count) {
+			print_line("test");
+		}
```

Comment:
Remember to remove this debug print :slightly_smiling_face:

Reply:
oops